lib/routes/utility.js

```diff
@@ -0,0 +1,35 @@
+// == BSD2 LICENSE ==
+// Copyright (c) 2016, Tidepool Project
+//
+// This program is free software; you can redistribute it and/or modify it under
+// the terms of the associated License, which is identical to the BSD 2-Clause
+// License as published by the Open Source Initiative at opensource.org.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+// FOR A PARTICULAR PURPOSE. See the License for more details.
+//
+// You should have received a copy of the License along with this program; if
+// not, you can obtain one from Tidepool Project at tidepool.org.
+// == BSD2 LICENSE ==
+
+'use strict';
+var _ = require('lodash');
+
+module.exports = function() {
+  return {
+    /*
+     * get the date as an ISO 8601 string from any valid datetime YYYY-MM-DDTHH:mm:ss.sssZ string
+     *
+     * @param {string} datetime
+     * @return {string} ISO 8601 string or null
+     */
+    getISODate: function(datetime) {
+      if (_.isEmpty(datetime)) {
+        return null;
+      }
+      datetime = datetime.replace(' ', '+');
```

Comment:
This is what was happening in the case of UTC + timezones. Hence the bandaid solution

```
it('date, time and offset UTC', function(done) {
      var returnedDate = utility.getISODate('2016-12-22T16:27:10 13:00');
      expect(returnedDate).to.exist;
      expect(returnedDate).to.equal('2016-12-22T03:27:10.000Z');
      done();
    });
```